### PR TITLE
Added a movie and a podcast!

### PR DESCRIPTION
**Pull request by margarethogg**
Look, I am doing the thing!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,14 @@ I first read this when I was eight or nine and my mom was trying to introduce me
 
 ## Movies
 
+#### [The Abandoned](http://www.imdb.com/title/tt0475937/?ref_=nv_sr_1) (2006) *by [Nacho Cerdà](https://en.wikipedia.org/wiki/Nacho_Cerd%C3%A0)* [5.6]
+
+Categories: `horror` `supernatural` `psychological thriller`
+
+Mostly a ghost story, with a little bit of zombie and sci-fi trope in there. the Abandoned is tightly plotted, wonderfully acted, and visually stunning. Definitely worth a watch and a re-watch.  - [@margarethogg](https://github.com/margarethogg)
```
Comment (thejessleigh) [inline README.md]: Missed a capital letter at the beginning of the second sentence here.

File: README.md [hunk 2]
```diff
@@ -143,6 +143,14 @@ I first read this when I was eight or nine and my mom was trying to introduce me
 
 ## Movies
 
+#### [The Abandoned](http://www.imdb.com/title/tt0475937/?ref_=nv_sr_1) (2006) *by [Nacho Cerdà](https://en.wikipedia.org/wiki/Nacho_Cerd%C3%A0)* [5.6]
```
Comment (thejessleigh) [inline README.md]: I think we can remove the query string from the imdb url.

**Review by thejessleigh — CHANGES_REQUESTED** [review]

**Comment by thejessleigh** [comment]
I've never heard of The Magnus Archives. I'm excited to check that out!

**Comment by margarethogg** [comment]
@thejessleigh It should be fixed. (lol am I doing this right?) I was adding another one at the times so apologies.

**Review by thejessleigh — APPROVED** [review]

**Comment by thejessleigh** [comment]
@margarethogg looks awesome. Thanks!

**Pull request merged**